src/pages/basic_data/app_admin/central_import/DuplicateSearchFilterForm.tsx

```diff
@@ -0,0 +1,270 @@
+  display: 'flex',
+  flexDirection: 'column',
+  marginTop: '1rem',
+  marginBottom: '1rem',
+});
+
+const StyledFormElementWrapper = muiStyled('div')({
+  display: 'flex',
+  marginBottom: '1rem',
+});
+
+const StyledFormControlLabel = muiStyled(FormControlLabel)({
+  minWidth: '11rem',
+});
+
+//TODO: move out
+export interface DuplicateSearchFilters {
+  doi: string;
+  title: string;
+  author: string;
+  yearPublished: string;
+  issn: string;
+}
+
+//TODO: move out
+export const emptyDuplicateSearchFilter = {
+  doi: '',
+  title: '',
+  author: '',
+  yearPublished: '',
+  issn: '',
+};
+
+const StyledButtonWrapper = muiStyled('div')({
+  display: 'flex',
+  justifyContent: 'flex-start',
+  marginBottom: '1rem',
+  alignItems: 'center',
+});
+
+interface DuplicateSearchFilterFormProps {
+  publication: Registration;
+  setDuplicateSearchFilters: any;
+}
+
+export const DuplicateSearchFilterForm = ({
+  publication,
+  setDuplicateSearchFilters,
+}: DuplicateSearchFilterFormProps) => {
+  const { t } = useTranslation('basicData');
+
+  const initialSearchParams = {
+    doi: publication.entityDescription?.reference?.doi ?? '',
+    title: publication.entityDescription?.mainTitle ?? '',
+    author: publication.entityDescription?.contributors[0]?.identity.name ?? '',
+    issn: '',
+    yearPublished: '',
+    isDoiChecked: true,
+    isTitleChecked: false,
+    isAuthorChecked: false,
+    isIssnChecked: false,
+    isYearPublishedChecked: false,
+  };
+
+  return (
+    <Formik
+      initialValues={initialSearchParams}
+      onSubmit={(values) => {
+        setDuplicateSearchFilters({
+          doi: values.isDoiChecked ? values.doi : '',
+          title: values.isTitleChecked ? values.title : '',
+          author: values.isAuthorChecked ? values.author : '',
+          issn: values.isIssnChecked ? values.issn : '',
+          yearPublished: values.isYearPublishedChecked ? values.yearPublished : '',
+        });
+      }}>
+      {(formikProps: FormikProps<any>) => (
+        <Form>
+          <StyledFormWrapper>
+            <FormGroup>
+              <StyledFormElementWrapper>
+                <Field
+                  as={StyledFormControlLabel}
+                  type="checkbox"
+                  name="isDoiChecked"
+                  control={<Checkbox />}
+                  label={t('central_import.doi')}
+                  data-testid="duplicate-search-doi-checkbox"
+                  onClick={(event: any) => {
+                    if (event.target.value) {
+                      formikProps.setFieldValue('isTitleChecked', false);
+                      formikProps.setFieldValue('isAuthorChecked', false);
+                      formikProps.setFieldValue('isIssnChecked', false);
+                      formikProps.setFieldValue('isYearPublishedChecked', false);
+                    }
+                  }}
+                />
+                <Field name={'doi'}>
+                  {({ field }: FieldProps) => (
```

Comment:
varierer hvor god jeg er på det selv, men ofte kjekt å legge til type på feltene også
```suggestion
                  {({ field }: FieldProps<string>) => (
```

Reply:
ahh. nice forslag